app/models/amp/Amp.scala

```diff
@@ -15,64 +15,120 @@
  */
 
 package models.amp
+import java.time.LocalDateTime
 
 import config.ApplicationConfig
 import models.registrationprogress.{Completed, NotStarted, Section, Started}
+import play.api.libs.json._
+import play.api.mvc.Call
 import typeclasses.MongoKey
 import uk.gov.hmrc.http.cache.client.CacheMap
 
-case class Amp(
-                hasChanged: Boolean = false,
-                hasAccepted: Boolean = false
-              ) {
+final case class Amp(
+                       id: String,
```

Comment:
My thinking was to keep the JSON we store in AMLS FE Mongo exactly the same as is stored in AMP Mongo. This could be subject to change once I get this wired up.

Reply:
ok